tools/README.md

```diff
@@ -0,0 +1,10 @@
+# Tools
+
+## Moonbeam-Launch
+
+Moonbeam-launch is a fork of https://github.com/paritytech/polkadot-launch that allows to run
+a test script on multiple nodes.
+
+To use it, follow instructions here to build polkadot in the same repo : https://github.com/PureStake/polkadot-launch/tree/moonbeam-launch
```

Comment:
This part could be improved slightly.

1. How do I know what version of Polkadot to build? At minimum mention to check in the Cargo.lock file. Maybe a script to build polkadot would be worthwhile too?
2. Allow specifying a path to polkadot (on CLI or by env variable or such). This allows using the already built polkadot I have in my normal polkadot repo.

Reply:
Oh, actually I see the path in the config file, so maybe just mention you can change the path there.

Reply:
>How do I know what version of Polkadot to build? 

This is indicated in the link, but I will add it to the README

>Maybe a script to build polkadot would be worthwhile too?

This feature will be added later and will probably be using docker

>so maybe just mention you can change the path there.

sg

Btw, now that parachain and standalone have been unified, `cargo build --release` needs to be run at the root of moonbeam repo, right?